### PR TITLE
various fixes

### PR DESCRIPTION
**Pull request by kil0byt3**
- add attachTo points for GBC180 ViV truck (3)
- add attachTo points for nh90 cargo variant (4)
- add attachTo points for AMF big plane I forget the name (8)
- added HEMTT Flatbed (Pacific) to preset, as a catch-all just in case 
	- adjusted attachTo points for HEMTT Flatbed (Pacific). Added one more (4 total), adjusted Y coord (they were floating)
	- prices were adjusted higher to match the extra carry space

- add GBC180 ammo truck and repair truck to preset instead of HEMTT, also added to the AI's list for rearm/repair
- same as above but with CUP MTVR for fuel (AMF doesn't seem to have a fuel truck)

- add CUP CH47F as a heavy-lift helo (because it already has attachTo points set up ha)

- added objectInits to: 
	- hide cover on GBC180 ViV truck (note: ViV variant only) to make a "pseudo-flatbed"
	- set CUP MTVR camo to be a more fitting woodland, it ain't perfect

- added various Boxloader objects to buy list to ease logistics (fuel barrels, empty box, 20ft iso, master pallet)

	- note that I did not test pretty much anything, just going to push this so you can merge, then we can both test at some point

### CODE REVIEW

File: presets/blufor/custom.sqf [hunk 1]
```diff
@@ -87,8 +87,8 @@ infantry_units = [
 
 light_vehicles = [
     ["B_T_Quadbike_01_F",25,0,25],                                      // Quad Bike
-    ["AMF_GBC180_PLATEAU_01",50,0,50],                                  // GBC 180 (Flatbed)
```
Comment (AChesheireCat) [inline presets/blufor/custom.sqf:90]: Is this line removing the gbc flatbed? It should be fine if we just keep it in the stuff, right?

**Comment by AChesheireCat** [comment]
otherwise, lgtm

**Review by AChesheireCat — CHANGES_REQUESTED** [review]

File: presets/blufor/custom.sqf (hunk 1 above)
Comment (kil0byt3) [reply presets/blufor/custom.sqf:90]: yes either way, it just doesn't work

**Comment by kil0byt3** [comment]
also my `objectInit` changes do, in fact, work 
image

image

**Pull request merged**